common/network-common/src/test/java/org/apache/spark/network/protocol/MessageWithHeaderSuite.java

```diff
@@ -150,7 +150,7 @@ private ByteBuf doWrite(MessageWithHeader msg, int minExpectedWrites) throws Exc
 
     @Override
     public long count() {
-      return 8 * writeCount;
+      return 8L * writeCount;
```

Comment:
The idea here is that 8 * writeCount is int * int, which could overflow, even though the result is meant to be long. Cast before the multiply. Probably not actually a bug in practice but who knows.